cartography/intel/gsuite/api.py

```diff
@@ -193,6 +193,7 @@ def load_gsuite_users(
         u.user_id = user.id,
         u.firstseen = $UpdateTag
         SET
+        u:UserAccount,
```

Comment:
In the data model itself, we could introduce a special field for "OntologyLabel". Would this help?